### PR TITLE
translate tutorial/part one

### PR DESCRIPTION
**Pull request by SamuelAlev**

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -1,190 +1,190 @@
...
+title: Apprenez à connaître blocs de construction de Gatsby
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-In the [**previous section**](/tutorial/part-zero/), you prepared your local development environment by installing the necessary software and creating your first Gatsby site using the [**“hello world” starter**](https://github.com/gatsbyjs/gatsby-starter-hello-world). Now, take a deeper dive into the code generated by that starter.
+Dans la [**section précédente**](/tutorial/part-zero/), vous avez préparé votre environnement de développement local en installant les logiciels nécessaires et créé votre premier site Gatsby en utilisant le [**starter “hello world”**](https://github.com/gatsbyjs/gatsby-starter-hello-world). Maintenant, plongeons dans le code généré par le starter.
 
-## Using Gatsby starters
+## Utilisation des starters Gatsby
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md:9]: Not translating starter is a great idea!

File: docs/tutorial/part-one/index.md [hunk 2]
```diff
@@ -1,190 +1,190 @@
...
 ```
 
-If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero.
+Si vous oubliez l'URL à la fin, Gatsby générera automatiquement un site pour vous en fonction du [**starter par défaut**](https://github.com/gatsbyjs/gatsby-starter-default). Dans cette section du tutoriel, restez avec le site “Hello World” que vous avez déjà créé dans la partie zéro.
 
-### ✋ Open up the code
+### ✋ Ouvrez le code
 
-In your code editor, open up the code generated for your “Hello World” site and take a look at the different directories and files contained in the ‘hello-world’ directory. It should look something like this:
+Dans votre éditeur, ouvrez le code généré pour votre site “Hello World” et prenez connaissance des différents répertoires et fichiers contenu dans le projet ‘hello-world’. Ça devrait ressembler a quelque chose comme ça :
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
Dans votre éditeur, ouvrez le code généré pour votre site “Hello World” et prenez connaissance des différents répertoires et fichiers contenus dans le projet ‘hello-world’. Ça devrait ressembler à quelque chose comme ça :
```

File: docs/tutorial/part-one/index.md [hunk 3]
```diff
@@ -1,190 +1,190 @@
...
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="./02-demo-hot-reloading.mp4"></source>
-  <p>Sorry! Your browser doesn't support this video.</p>
+  <p>Désolé ! Votre navigateur ne supporte pas cette vidéo.</p>
 </video>
 
-> 💡 Gatsby uses **hot reloading** to speed up your development process. Essentially, when you’re running a Gatsby development server, the Gatsby site files are being “watched” in the background — any time you save a file, your changes will be immediately reflected in the browser. You don’t need to hard refresh the page or restart the development server — your changes just appear.
+> 💡 Gatsby utilise le **rechargement immédiat** pour accélerer votre processus de développement. Lorsque vous exécutez un serveur de développement Gatsby, les fichiers du site Gatsby sont “surveillés” en arrière-plan - chaque fois que vous enregistrez un fichier, vos modifications sont immédiatement reflétées dans le navigateur. Vous n'avez pas besoin d'actualiser la page ou de redémarrer le serveur de développement, vos modifications apparaissent.
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
> 💡 Gatsby utilise le **rechargement immédiat** pour accélérer votre processus de développement. Lorsque vous exécutez un serveur de développement Gatsby, les fichiers du site Gatsby sont “surveillés” en arrière-plan - chaque fois que vous enregistrez un fichier, vos modifications sont immédiatement reflétées dans le navigateur. Vous n'avez pas besoin d'actualiser la page ou de redémarrer le serveur de développement, vos modifications apparaissent.
```

File: docs/tutorial/part-one/index.md [hunk 4]
```diff
@@ -1,190 +1,190 @@
...
+![Ajout d'une image](04-add-image.png)
 
-### Wait… HTML in our JavaScript?
+### Attendez… De l'HTML dans notre JavaScript?
 
-_If you’re familiar with React and JSX, feel free to skip this section._ If you haven’t worked with the React framework before, you may be wondering what HTML is doing in a JavaScript function. Or why we’re importing `react` on the first line but seemingly not using it anywhere. This hybrid “HTML-in-JS” is actually a syntax extension of JavaScript, for React, called JSX. You can follow along with this tutorial without prior experience with React, but if you’re curious, here’s a brief primer…
+_Si vous êtes familier avec React et JSX, n'hésitez pas alors à passer cette section.._ Si vous n’avez jamais utilisé le framework React auparavant, vous vous demandez peut-être ce que HTML fait dans une fonction JavaScript. Ou pourquoi nous importons `react` sur la première ligne mais ne l’utilisons apparemment pas nulle part. Cet façon hybride «HTML-in-JS» est en fait une extension de syntaxe de JavaScript pour React, appelée JSX. Vous pouvez suivre ce tutoriel sans expérience préalable de React, mais si vous êtes curieux, voici un bref aperçu…
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
_Si vous êtes familier avec React et JSX, n'hésitez pas alors à passer cette section.._ Si vous n’avez jamais utilisé le framework React auparavant, vous vous demandez peut-être ce que HTML fait dans une fonction JavaScript. Ou pourquoi nous importons `react` sur la première ligne, mais ne l’utilisons apparemment pas nulle part. Cette façon hybride «HTML-in-JS» est en fait une extension de syntaxe de JavaScript pour React, appelée JSX. Vous pouvez suivre ce tutoriel sans expérience préalable de React, mais si vous êtes curieux, voici un bref aperçu…
```

File: docs/tutorial/part-one/index.md [hunk 5]
```diff
@@ -1,190 +1,190 @@
...
+En JavaScript pur, cela ressemble plus à ceci:
 
 ```javascript:title=src/pages/index.js
 import React from "react"
 
 export default () => React.createElement("div", null, "Hello world!")
 ```
 
-Now you can spot the use of the `'react'` import! But wait. You’re writing JSX, not pure HTML and JavaScript. How does the browser read that? The short answer: It doesn’t. Gatsby sites come with tooling already set up to convert your source code into something that browsers can interpret.
+Vous pouvez maintenant voir l’utilisation de l’importation `'react'`! Mais attendez. Vous écrivez en JSX, et non pas en HTML pur et en JavaScript. Comment le navigateur lit-il cela? La réponse courte: ce n’est pas le cas. Les sites Gatsby sont fournis avec des outils déjà configurés pour convertir votre code source en quelque chose que les navigateurs peuvent interpréter.
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
Vous pouvez maintenant voir l’utilisation de l’import `'react'`! Mais attendez. Vous écrivez en JSX, et non pas en HTML pur et en JavaScript. Comment le navigateur lit-il cela? La réponse courte: ce n’est pas le cas. Les sites Gatsby sont fournis avec des outils déjà configurés pour convertir votre code source en quelque chose que les navigateurs peuvent interpréter.
```

File: docs/tutorial/part-one/index.md [hunk 6]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
+Après avoir enregistré le fichier, vous devriez voir la page de contact et être capable de naviguer entre celle-ci et la page d'accueil.
 
 <video controls="controls" loop="true">
   <source type="video/mp4" src="./10-linking-between-pages.mp4"></source>
-  <p>Sorry! You browser doesn't support this video.</p>
+  <p>Désolé ! Votre navigateur ne supporte pas cette vidéo.</p>
 </video>
 
-The Gatsby `<Link />` component is for linking between pages within your site. For external links to pages not handled by your Gatsby site, use the regular HTML `<a>` tag.
+Le component Gatsby `<Link />` permet de lier différentes pages de votre site. Pour les liens externes non géré par votre site Gatsby, utilisez la balise HTML par défaut `<a>`.
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
Le composant Gatsby `<Link />` permet de lier différentes pages de votre site. Pour les liens externes non gérés par votre site Gatsby, utilisez la balise HTML par défaut `<a>`.
```

File: docs/tutorial/part-one/index.md [hunk 7]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
 
-In this section you:
+Dans cette section, vous :
 
-- Learned about Gatsby starters, and how to use them to create new projects
-- Learned about JSX syntax
-- Learned about components
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+- Avez appris sur les générateurs de Gatsby, et comment les utiliser pour créer des nouveaux projets
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
- Avez découvert les starters de Gatsby, et comment les utiliser pour créer des nouveaux projets
```

File: docs/tutorial/part-one/index.md [hunk 8]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
+- Avez appris sur la syntaxe de JSX
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
- Avez découvert la syntaxe de JSX
```

File: docs/tutorial/part-one/index.md [hunk 9]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
+- Avez appris sur les composants
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
- Avez découvert les composants
```

File: docs/tutorial/part-one/index.md [hunk 10]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
+- Avez appris sur les composants et sous-composants de pages Gatsby
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
- Avez découvert les composants et sous-composants de pages Gatsby
```

File: docs/tutorial/part-one/index.md [hunk 11]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
+- Avez appris sur les “props” React et la réutilisation de composants React
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
- Avez découvert les “props” React et la réutilisation de composants React
```

File: docs/tutorial/part-one/index.md [hunk 12]
```diff
@@ -319,70 +318,69 @@ export default () => (
...
 
-Now, move on to [**adding styles to your site**](/tutorial/part-two/)!
+Maintenant, continuez avec [**ajouter du styles à votre site**](/tutorial/part-two/)!
```
Comment (dantereve) [inline docs/tutorial/part-one/index.md]:
```suggestion
Maintenant, continuez avec [**ajouter des styles à votre site**](/tutorial/part-two/)!
```

**Review by dantereve — CHANGES_REQUESTED** [review]
Good job 🥳 ! I've just spotted some typos and did some suggestions to fix them.

**Comment by SamuelAlev** [comment]
Thank you for your feedback !
I fixed the typos.

**Review by dantereve — APPROVED** [review]

**Pull request merged**